scripts/issue/732/promote_aura_vaults.py

```diff
@@ -0,0 +1,32 @@
+from brownie import interface
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+from rich.console import Console
+
+C = Console()
+
+TECH_OPS = registry.eth.badger_wallets.techops_multisig
+DEV = registry.eth.badger_wallets.dev_multisig
+REGISTRY = registry.eth.registry_v2
+VAULTS = registry.eth.sett_vaults
+STATUS = 1 # Guarded
```

Comment:
```
  enum VaultStatus {
    deprecated,
    experimental,
    guarded,
    open
  }
  ```

this should be status: 2 

i believe ser

Reply:
Good catch, you are right. Changing..